lib/matplotlib/tests/test_legend.py

```diff
@@ -522,3 +522,32 @@ def test_legend_title_empty():
     leg = ax.legend()
     assert leg.get_title().get_text() == ""
     assert leg.get_title().get_visible() is False
+
+
+@image_comparison(baseline_images=['legend_large_markers'],
+                  extensions=['png'])
```

Comment:
Ooops, somehow managed to lose this comment:  These should be style='mpl20' if possible.  You need the text, but you could clear the x and y ticks to reduce the amount of text.  We are trying to make image tests have less text to make it easier for them to pass different FreeType versions.